server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertPlan.java

```diff
@@ -33,7 +34,7 @@
 import java.util.List;
 import java.util.Set;
 
-public abstract class InsertPlan extends PhysicalPlan {
+public abstract class InsertPlan extends PhysicalPlan implements WALEditValue {
```

Comment:
Why implements WALEditValue?

Reply:
Use WALEditValue to manage InsertPlan, DeletePlan and IMemTable, you can see more details in the WALEdit class.